scripts/docker/ubuntu.16.04/Dockerfile

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+FROM ubuntu:16.04
+
+# Install the base toolchain we need to build anything (clang, cmake, make and the like)
```

Comment:
In this particular case, they are actually somewhat different. Plus, the version of packages changes  according to the distro and the version of the distro that you use.